Makefile

```diff
@@ -22,6 +22,7 @@ buildir:
 # General targets (Public)
 .PHONY: clean
 clean: ##@STATES Delete cluster of type $BACKEND and location build$CLUSTER_NAME
+clean: scf-clean # first scf-clean because of external-dns annotations
```

Comment:
Just a side note: This is also needed (at least for AWS EKS) to ensure that the kubecf PVC are deleted when tearing things down. Just doing `clean` leaves them active if unused, and they keep accumulating costs. So +1 on this piece of the whole.

Reply:
True, reworded comment :)